README.rst

```diff
@@ -1,3 +1,13 @@
+.. image:: https://travis-ci.com/PyCQA/doc8.svg?branch=master
+    :target: https://travis-ci.com/PyCQA/doc8
+.. image:: https://img.shields.io/pypi/v/doc8
+    :alt: PyPI
+    :target: https://pypi.org/project/doc8/
+.. image:: https://img.shields.io/pypi/l/doc8
+    :alt: PyPI - License
+.. image:: https://img.shields.io/github/last-commit/pycqa/doc8
+    :alt: GitHub last commit
```

Comment:
Small thing, but any chance we could put these after the README title?

Reply:
This will break display of readme on PYPI and also on github, I think that 9/10 projects using badges do display them before the header line, otherwise the rendered page looks weird.

See https://github.com/tox-dev/tox as an example.

Reply:
Break display? Okay, I'll trust you on this so. This was a style thing anyway